README.rst

```diff
@@ -0,0 +1,11 @@
+Robpy
+=========
+
+**Robpy** is a Python module for robust statistical methods built on top of scikit-learn and
+SciPy. 
+
+The project was started in 2023 by Tim Verdonck, Jakob Raymaekers, Thomas Servotte, Sarah Leyder & Thomas Decorte at the University of Antwerp and imec IDLab. 
+The project is currently maintained by a the same team and distributed under MIT license.
```

Comment:
The project is currently maintained by ~a~ the same team and distributed under MIT license.